pkg/openstack/glance.go

```diff
@@ -18,8 +19,11 @@ import (
 
 const (
 	// svcSelector is used as selector to get the list of "Services" associated
-	// to a specific glanceAPI instance
-	svcSelector = "glanceAPI"
+	// to a specific glanceAPI instance. It must be different from glanceAPI
+	// label set by the service operator as in case of split glanceAPI type the
+	// the label on public svc gets set to -external and internal instance svc
+	// to -internal instead of the glance top level glanceType split
+	svcSelector = "tlGlanceAPI"
```

Comment:
thats intended and should say something like top level glanceAPI. it is needed for the glance split use case, since the glance-operator will set `glanceAPI: external` and `glanceAPI: internal` except of `glanceAPI: split` which is applied at the top level. otherwise the services used for the split glanceAPI can not be identified properly and connect to the split glanceAPI config